app/controllers/users_controller.rb

```diff
@@ -9,9 +9,23 @@ def create
     @user = User.new(user_params)
     if @user.save
       handle_invitation
-      AppMailer.delay.send_welcome_message(@user)
-      session[:user_id] = @user.id
-      redirect_to home_path, notice: "You are signed in"
+      Stripe.api_key = ENV['STRIPE_SECRET_KEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.